fabtests/functional/inj_complete.c

```diff
@@ -181,12 +190,23 @@ int main(int argc, char **argv)
 		case 'v':
 			opts.options |= FT_OPT_VERIFY_DATA;
 			break;
+		case 'A':
+			if (!strncasecmp("inj_complete", optarg, 12)) {
+				flag = FI_INJECT_COMPLETE;
+			} else if (!strncasecmp("inject", optarg, 6)) {
+				flag = FI_INJECT;
+			} else {
+				printf ("Unsupported flag\n");
+				return EXIT_FAILURE;
```

Comment:
I would consider renaming the test to maybe fi_inject_test.  The current name aligns with testing for inject_complete semantics.

I would go even further and just have the test verify both options and avoid needing to pass in the parameters.  We should look at doing this for other tests, so that running a test can validate more of the API without needing to re-run the test with a different set of options.

Reply:
- I renamed it to fi_msg_inject and also renamed the file to fi_msg_inject.c as I wanted to keep the names similar. Is that ok or would you want me to rename the test to fi_inject_test? 

- I was trying to avoid the need to pass a parameter for FI_INJECT and FI_INJECT_COMPLETE but I found out that non sendmsg path that requires the EP to be configured with OP flags so it would be configured with both and so iterating through all flags would require the test to re-initialize to test each case. So I went with the current way.